controllers/api.controller.js

```diff
@@ -17,6 +18,17 @@ exports.getTopics = (req, res, next) => {
     });
 };
 
+exports.delComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  deleteCommentById(comment_id)
+    .then(() => {
+      res.status(204).send({ status: 204, msg: "no content" });
```

Comment:
Because the status code is the only thing that is required (no content will be sent by express) we can just invoke .send